app/assets/javascripts/bastion/components/notification.service.js

```diff
@@ -16,8 +16,15 @@ angular.module('Bastion.components').service("Notification", ['$interpolate', 'f
         return result;
     }
 
-    this.setSuccessMessage = function (message, context) {
-        foreman.toastNotifications.notify({message: interpolateIfNeeded(message, context), type: 'success'});
+    this.setSuccessMessage = function (message, options) {
+        /* eslint-disable no-unused-expressions, angular/ng_definedundefined*/
```

Comment:
You can use `angular.isUndefined`, no need to disable it, it should function the same as `undefined`